tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -1680,10 +1680,25 @@ bool CUDABlas::DoBlasGemm(
   return DoBlasInternal(
       dynload::cublasSgemmEx, stream, true /* = pointer_mode_host */,
       CUDABlasTranspose(transa), CUDABlasTranspose(transb), m, n, k, &alpha,
-      CUDAMemory(a), CUBLAS_DATA_HALF, lda,
-      CUDAMemory(b), CUBLAS_DATA_HALF, ldb,
-      &beta,
-      CUDAMemoryMutable(c), CUBLAS_DATA_HALF, ldc);
+      CUDAMemory(a),
```

Comment:
yes or I can also `#define CUBLAS_DATA_HALF CUDA_R_16F` for `CUDA_VERSION >= 8000` which might make the code look a bit cleaner...